modules/ml/src/test/java/org/apache/ignite/ml/naivebayes/discrete/DiscreteNaiveBayesTrainerTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ignite.ml.naivebayes.discrete;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.ignite.ml.common.TrainerTest;
+import org.apache.ignite.ml.dataset.impl.local.LocalDatasetBuilder;
+import org.apache.ignite.ml.math.primitives.vector.VectorUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/** Test for {@link DiscreteNaiveBayesTrainer} */
+public class DiscreteNaiveBayesTrainerTest extends TrainerTest {
+
+    /** Precision in test checks. */
+    private static final double PRECISION = 1e-2;
+    /** */
+    private static final double LABEL_1 = 1.;
+    /** */
+    private static final double LABEL_2 = 2.;
+
+    /** Data. */
+    private static final Map<Integer, double[]> data = new HashMap<>();
+    /** */
+    private static final double[][] thresholds = new double[][] {{.5}, {.5}, {.5}, {.5}, {.5}};
```

Comment:
you always use same threshold values with just one bound
may you add several tests with other threshold combinations?

Reply:
I added a test with a few different thresholds.